### PR TITLE
Fall back to the default Sublime Text Expand to Scope when if

### PR DESCRIPTION
**Pull request by evandrocoan**
cannot expand anymore. Issue: https://github.com/aronwoost/sublime-expand-region/issues/75

### CODE REVIEW

File: ExpandRegion.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def run(self, edit, language="", undo=False, debug=True):
       print("Determined language: '{0}'".format(language))
 
     new_regions = []
+    call_sublime_expand_scope = False
```
Comment (aronwoost) [inline ExpandRegion.py]: Improve naming. Maybe `expand_region_done`? Or `did_expand_region`?

File: ExpandRegion.py [hunk 2]
```diff
@@ -81,18 +82,25 @@ def run(self, edit, language="", undo=False, debug=True):
...
-    # replace the selections with the new regions
-    view.sel().clear()
-    for sel in new_regions:
-      view.sel().add(sel)
-
-    settings = sublime.load_settings("ExpandRegion.sublime-settings")
-    do_force_enable_soft_undo = settings.get("force_soft_undo_integration")
-    if do_force_enable_soft_undo:
-      _force_enable_soft_undo(view, edit, new_regions)
+        # # if there is no result, keep the current region
```
Comment (aronwoost) [inline ExpandRegion.py]: Remove double `#`

File: ExpandRegion.py [hunk 3]
```diff
@@ -81,18 +82,25 @@ def run(self, edit, language="", undo=False, debug=True):
...
+        # new_regions.append(region)
```
Comment (aronwoost) [inline ExpandRegion.py]: What is the effect when this is commended out?

Also: remove dead code instead of comment.

File: ExpandRegion.py (hunk 2 above)
Comment (aronwoost) [reply ExpandRegion.py]: Also: does the comment still make sense here?

File: ExpandRegion.py [hunk 4]
```diff
@@ -81,18 +82,25 @@ def run(self, edit, language="", undo=False, debug=True):
...
+        call_sublime_expand_scope = True
+
+    if call_sublime_expand_scope:
+      #  "command": "expand_selection", "args": {"to": "scope"}, "caption": "Expand Selection to Scope" }
```
Comment (aronwoost) [inline ExpandRegion.py]: Remove

File: ExpandRegion.py (hunk 1 above)
Comment (evandrocoan) [reply ExpandRegion.py]: I inverted the name to `is_region_expanded`.

File: ExpandRegion.py (hunk 2 above)
Comment (evandrocoan) [reply ExpandRegion.py]: Not much, I changed the comment to incorporate new changes.

File: ExpandRegion.py (hunk 3 above)
Comment (evandrocoan) [reply ExpandRegion.py]: When this is commented out we cannot recreate the same regions, however when falling back to Sublime Text built-in command, we do not need to recreate regions. Also, I am sure why before the regions were being recreated when there was no new regions. It seems redundant to recreate them when no new regions are being added.

File: ExpandRegion.py (hunk 4 above)
Comment (evandrocoan) [reply ExpandRegion.py]: This I forgot to remove. Put it there to write the command and forget about removing it.

**Comment by evandrocoan** [comment]
@aronwoost, I pushed a new commit with the changes.

**Comment by aronwoost** [comment]
@evandrocoan Awesome! Can you provide me with a text snipped so I can test the expected behavior?

**Comment by evandrocoan** [comment]
The only snippet I have are is when using it on a simple latex file like:
```
    \section{text}

    On am we offices expense thought. Its hence ten smile age means. Seven chief sight far point any. Of so high into easy. Dashwoods eagerness oh extensive as discourse sportsman frankness. Husbands see disposed surprise likewise humoured yet pleasure. Fifteen no inquiry cordial so resolve garrets as. Impression was estimating surrounded solicitude indulgence son shy.     
```

And I try to expand work on this text. Other than these situations, the Sublime Text build in expansion does not behave nicely.

**Comment by aronwoost** [comment]
Thank you! 👍

**Pull request merged**